radio/src/myeeprom.h

```diff
@@ -60,7 +60,11 @@
 #endif
 
 #define HAS_ENABLE_PARAM(func)         ((func) < FUNC_FIRST_WITHOUT_ENABLE || (func == FUNC_BACKLIGHT))
-#define HAS_REPEAT_PARAM(func)         (IS_PLAY_FUNC(func) || IS_HAPTIC_FUNC(func))
+#if defined(COLORLCD)
```

Comment:
Your changes depend on COLORLCD here, but on LIBOPENUI in functions.cpp. please use the same everywhere. I know that it is not very consistent right now, but we can try to get better.

Reply:
Done